.eslintrc.js

```diff
@@ -25,6 +26,7 @@ module.exports = {
       files: ['*.js', '*.mjs', '*.ts', '*.tsx'],
       rules: {
         semi: [2, 'never'],
+        'react-native/no-inline-styles': 0,
```

Comment:
We have a lot of these and imo it's fine. Eventually we'll have a proper design system too.